layer1/ethereum/contracts.go

```diff
@@ -128,13 +126,15 @@ func (c *Contracts) lookupContracts() error {
 		logAndEat(logger, err)
 
 		// ALCB
-		c.alcbAddress, err = lookup("ALCB")
-		logAndEat(logger, err)
-		// TODO: bring back this check once we re-deploy BToken
+		// TODO: bring it back once we deploy ALCB
+		// c.alcbAddress, err = lookup("ALCB")
+		// logAndEat(logger, err)
 		// if bytes.Equal(c.alcbAddress.Bytes(), make([]byte, 20)) {
 		// 	continue
 		// }
-
+		// workaround for now, just putting a random address
```

Comment:
This is a bit vague; someone looking at the code won't know what this workaround is for - perhaps add a link to an issue/bug that was raised where people can get a bit more info/context

Reply:
addressed!